code/mob/new_player.dm

```diff
@@ -385,7 +385,14 @@ mob/new_player
 					logTheThing("debug", character, null, "<b>Late join:</b> added player to ticker.minds.")
 					ticker.minds += character.mind
 				logTheThing("debug", character, null, "<b>Late join:</b> assigned job: [JOB.name]")
-
+				//latejoin participation stuff
+#ifdef RP_MODE
+				var/max_participate_time = 80 MINUTES
+#endif
+				var/max_participate_time = 40 MINUTES
```

Comment:
this will compile error on rp mode.
this is what u want probly:
```suggestion
        var/max_participate_time = 40 MINUTES
#ifdef RP_MODE
				max_participate_time = 80 MINUTES
#endif
```

Reply:
oo thats smart zew, thanks. duplicate definition, right?

Reply:
yup